assets/src/edit-story/components/panels/document/slug/test/slug.js

```diff
@@ -77,30 +77,24 @@ describe('SlugPanel', () => {
     expect(url).toBeInTheDocument();
   });
 
-  it('should allow trailing spaces while typing but not onblur', async () => {
+  it('should allow now allow trailing spaces while typing or onblur', async () => {
```

Comment:
^ There seems to be 1 "allow" too many now 😄

Reply:
shoooot, nice catch